config/chime_science_run_gpu.yaml

```diff
@@ -458,6 +456,17 @@ gpu_rfi_output_buffers:
   gpu_rfi_output_buffer_3:
     kotekan_buffer: standard
 
+gpu_rfi_valve_buffers:
+  num_frames: buffer_depth
```

Comment:
Frame size is missing here.

Reply:
Yeah. I found this one out pretty rudely last night.